app/workers/queue_listener.rb

```diff
@@ -28,6 +30,10 @@ def create_action(params)
     AkActionCreator.create_action(params[:params][:slug], params[:params][:body])
   end
 
+  def update_resource(params)
+    Ak::Updater.update(uri: params[:uri], body: params[:params])
```

Comment:
I want to namespace the whole API with `AK`, and this is my first step. The rest will be refactored in another PR.